osgp/platform/osgp-secret-management/src/integration-test/resources/test-requests/activateSecrets.xml

```diff
@@ -1,7 +1,15 @@
-<ns:activateSecretsRequest xmlns:ns="http://www.opensmartgridplatform.org/schemas/security/secretmanagement">
+<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"
+               xmlns:ns="http://www.opensmartgridplatform.org/schemas/security/secretmanagement">
+  <soap:Header>
+    <ns:correlationUid>123456</ns:correlationUid>
+  </soap:Header>
+  <soap:Body>
+  <ns:activateSecretsRequest xmlns:ns="http://www.opensmartgridplatform.org/schemas/security/secretmanagement">
   <ns:DeviceId>E0000000000000000</ns:DeviceId>
   <ns:SecretTypes>
     <ns:SecretType>E_METER_AUTHENTICATION_KEY</ns:SecretType>
     <ns:SecretType>E_METER_ENCRYPTION_KEY_UNICAST</ns:SecretType>
   </ns:SecretTypes>
-</ns:activateSecretsRequest>
\ No newline at end of file
+</ns:activateSecretsRequest>
```

Comment:
The indentation looks a bit messy after the change to include the SOAP envelope with the sample test request.
Would be nice to end these text based files with a newline character at the end of the last line. That would avoid the "No newline at end of file" symbols in the GitHub diffs, and lead to nicer - more readable - default behavior whenever the contents of a number of these files would be printed somewhere one after the other.

(Similar comments could be made with a number of the other altered test requests.)

Reply:
Added newline at the end of the file and applied general formatting for all test xml files